utils/v10.ts

```diff
@@ -101,3 +107,53 @@ export function isLinkButton(component: APIButtonComponent): component is APIBut
 export function isInteractionButton(component: APIButtonComponent): component is APIButtonComponentWithCustomId {
 	return component.style !== ButtonStyle.Link;
 }
+
+// Message Components
+
+/**
+ * A type-guard check for button message component interactions
+ * @param interaction The message component interaction to check against
+ * @returns A boolean that indicates if the message component is a button
+ */
+export function isMessageComponentButtonInteraction(
```

Comment:
Can you also add a generic isMessageComponentInteraction? 🙏

Reply:
This is done